scripts/run_gesture_recognition.py

```diff
@@ -7,46 +7,66 @@
                              [--path_in=FILENAME]
                              [--path_out=FILENAME]
                              [--title=TITLE]
+                             [--model_name=NAME]
+                             [--model_version=VERSION]
                              [--use_gpu]
   run_gesture_recognition.py (-h | --help)
 
 Options:
   --path_in=FILENAME         Video file to stream from
   --path_out=FILENAME        Video file to stream to
   --title=TITLE              This adds a title to the window display
+  --model_name=NAME          Name of the model to be used.
+  --model_version=VERSION    Version of the model to be used.
+  --use_gpu                  Whether to run inference on the GPU or not.
```

Comment:
should we default to one model version ?

Reply:
It's not necessary:
> When these arguments are not used (default behavior), it will loop over SUPPORTED_MODEL_CONFIGURATIONS, from top to bottom, until it can find a model configuration for which the user has all necessary files

Adding a default model_name/model_version wouldn't add anything